example/README.md

```diff
@@ -0,0 +1,224 @@
+# Deploying Hyperlane with Osmosis Testnet
```

Comment:
Not super intuitive that this is in the example folder?

Reply:
We can move it to /docs or somewhere else